app/scripts/background.js

```diff
@@ -561,6 +573,11 @@ async function initialize() {
     await sendReadyMessageToTabs();
     log.info('MetaMask initialization complete.');
 
+    // Set initial idle state after initialization is complete
+    extensionUpdateManager.setIdleState(true);
```

Comment:
The UI may be open already by this point. Perhaps we should check rather than assuming the extension is idle?

Reply:
I added the ui check 9d5459a3bc355a16a7c3b0ba67e9df03e94b5b0e. Open to make it default as `true` as well. the latter could be a cleaner approach